docs/src/components/Layout/FrameworkChooser.tsx

```diff
@@ -1,66 +1,50 @@
-import {
-  Image,
-  ToggleButton,
-  ToggleButtonGroup,
-  VisuallyHidden,
-} from '@aws-amplify/ui-react';
+import { Flex, Image, Button, useTheme } from '@aws-amplify/ui-react';
 
+import { capitalize } from 'lodash';
 import { useCustomRouter } from '@/components/useCustomRouter';
+import Link from 'next/link';
 import { FRAMEWORKS } from '@/data/frameworks';
 
-export const FrameworkChooser = ({ platform }) => {
-  const { replace, pathname } = useCustomRouter();
+interface FrameworkLinkProps extends FrameworkChooserProps {
+  platform: string;
+}
 
-  const chooseFramework = (platform) => {
-    const { hash } = window.location;
-    replace(
-      {
-        hash,
-        pathname: pathname === '/' ? '/[platform]' : pathname,
-        query: { platform },
-      },
-      // `as?` prop  isn't needed when URL is already provided
-      undefined,
-      {
-        // Scroll to top if a new page
-        scroll: hash ? false : true,
-      }
-    );
+const FrameworkLink = ({ platform, onClick }: FrameworkLinkProps) => {
+  const { pathname, query } = useCustomRouter();
 
-    // Because layout may change, explicitly tell the browser to scroll to that anchor
-    // e.g. <a id="#variation" />
-    if (hash) {
-      document.getElementById(hash.slice(1)).scrollIntoView();
-    }
-  };
+  const isCurrent = query.platform === platform;
+  const classNames = `docs-framework-link ${isCurrent ? 'current' : ''}`;
+  const href = pathname.includes('[platform]')
+    ? pathname.replace('[platform]', platform)
+    : `/${platform}`;
 
   return (
-    <ToggleButtonGroup
-      value={platform}
-      size="small"
-      onChange={(value: string) => {
-        chooseFramework(value);
-      }}
-      isExclusive
-      isSelectionRequired
-    >
+    <Link href={href} passHref>
+      <Button as="a" size="small" className={classNames} onClick={onClick}>
+        <Image
+          alt=""
+          height="1.25rem"
+          width="1.25rem"
+          display="block"
+          src={`/svg/integrations/${platform}.svg`}
+        />
+        {capitalize(platform)}
+      </Button>
+    </Link>
+  );
+};
+
+interface FrameworkChooserProps {
+  onClick?: () => void;
```

Comment:
Is it optional? I thought we've already added it on L46?

```suggestion
  onClick: () => void;
```